external-import/mandiant/src/connector/base.py

```diff
@@ -646,13 +737,21 @@ def process_message(self):
                 self.helper.connector_logger.info("Connector stop")
                 sys.exit(0)
 
+            except StateError as err:
+                self.helper.connector_logger.error(str(err))
```

Comment:
Would it be interesting to clarify the message ?
```suggestion
                self.helper.connector_logger.error(
                    "Failed du to connector state error", {"error": str(err)}
                )
```

Reply:
Thanks for your suggestion !